.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -8,14 +8,21 @@ b) Framework functionality achievable only by the core.
 If this PR can be done as a 3rd-Party Community Add-On, we encourage that for quicker adoption.
 
 If you believe your functionality is valuable to the entire Qwik Community, discuss it in the Qwik Discord channels for potential inclusion in the core.
+
+First of all, make sure your PR title is descriptive and matches our commit title guidelines.
+
+Also make sure your PR follows all the guidelines in the [CONTRIBUTING.md](./CONTRIBUTING.md) document.
+
 -->
 
 # What is it?
 
-- [ ] Feature / enhancement
-- [ ] Bug
-- [ ] Docs / tests / types / typos
-- [ ] Infra
+<!-- pick one and remove the others -->
+
+- Feature / enhancement
+- Bug
+- Docs / tests / types / typos
+- Infra
```

Comment:
nit pick but I prefer the checkbox, a little more cumbersome to delete 3 lines 🙈

Reply:
But then we see all these PRs that are "incomplete" in the overview

Reply:
what do you mean? screenshot maybe?

Reply:
![image](https://github.com/user-attachments/assets/df3bd70e-800e-413f-88a7-625be0aada8b)

Reply:
It will also make the description less busy. Let's try it and see, we can still change it later.